Doc/howto/a-conceptual-overview-of-asyncio.rst

```diff
@@ -9,12 +9,11 @@ model of how :mod:`asyncio` fundamentally works, helping you understand the
 how and why behind the recommended patterns.
 
 You might be curious about some key :mod:`!asyncio` concepts.
-You'll be comfortably able to answer these questions by the end of this
-article:
+By the end of this article, you'll be able to answer these questions:
```

Comment:
I like the re-arrangement of this sentence. I'd prefer to add "comfortably" back. I think it offers some descriptiveness and supportiveness.